legal-api/src/legal_api/services/filings/validations/court_order.py

```diff
@@ -30,13 +31,24 @@ def validate(business: Business, court_order: Dict) -> Optional[Error]:
         return Error(HTTPStatus.BAD_REQUEST, [{'error': babel('A valid business and filing are required.')}])
     msg = []
 
-    if not get_str(court_order, '/filing/courtOrder/orderDetails'):
-        msg.append({'error': babel('Court Order is required.'), 'path': '/filing/courtOrder/orderDetails'})
-
     effect_of_order = get_str(court_order, '/filing/courtOrder/effectOfOrder')
     if effect_of_order and effect_of_order != 'planOfArrangement':
         msg.append({'error': babel('Invalid effectOfOrder.'), 'path': '/filing/courtOrder/effectOfOrder'})
 
+    file_key_path = '/filing/courtOrder/fileKey'
+    file_key = get_str(court_order, file_key_path)
+
+    order_details_path = '/filing/courtOrder/orderDetails'
+    order_details = get_str(court_order, order_details_path)
+
+    if not order_details and not file_key:
+        msg.append({'error': babel('Court Order is required (in orderDetails/fileKey).'), 'path': '/filing/courtOrder'})
```

Comment:
can we break this up and return separate validation errors for `order_details` and `file_key`?

Reply:
only one of it is required. how do we break up this?

Reply:
oh, i see.  forget my comment then :)